src/renderers/dom/shared/CSSPropertyOperations.js

```diff
@@ -236,6 +249,10 @@ var CSSPropertyOperations = {
     }
   },
 
+  multi: function() {
+    arguments.__isMultiCssValue = true;
+    return arguments;
```

Comment:
Isn’t leaking `arguments` bad in general?
https://github.com/petkaantonov/bluebird/wiki/Optimization-killers#32-leaking-arguments